EasyInventory/src/main/java/com/easyinventory/enterprise/dto/Item.java

```diff
@@ -0,0 +1,12 @@
+package com.easyinventory.enterprise.dto;
+
```

Comment:
For finalProject/EasyInventory/src/main/java/com/easyinventory/enterprise/model/Item.java I instead recommend you to you Lombok